Drawer/Drawer.js

```diff
@@ -61,6 +61,10 @@ const DrawerBase = kind({
 		className: 'drawer'
 	},
 
+	computed: {
+		role: () => 'alert'
+	},
```

Comment:
I think that it might be better to define the `role` not in the `computed` but in the `defaultProps`. By doing it, app developers could override it if needed.
```
defaultProps: {
  role: 'alert'
},
````

Reply:
I fixed it.